android/app/build.gradle

```diff
@@ -135,17 +136,29 @@ android {
         variant.outputs.each { output ->
             // For each separate APK per architecture, set a unique version code as described here:
             // https://developer.android.com/studio/build/configure-apk-splits.html
+            // Example: versionCode 1 will generate 1001 for armeabi-v7a, 1002 for x86, etc.
             def versionCodes = ["armeabi-v7a": 1, "x86": 2, "arm64-v8a": 3, "x86_64": 4]
             def abi = output.getFilter(OutputFile.ABI)
             if (abi != null) {  // null for the universal-debug, universal-release variants
                 output.versionCodeOverride =
-                        versionCodes.get(abi) * 1048576 + defaultConfig.versionCode
+                        defaultConfig.versionCode * 1000 + versionCodes.get(abi)
             }
         }
     }
 }
 
+repositories {
+  mavenCentral()
+}
+
 dependencies {
+    debugImplementation("com.facebook.flipper:flipper:${FLIPPER_VERSION}") {
```

Comment:
we are sure we wanna flipper ? is million reason for not - builds in bitrise and others

Reply:
Flipper is nice tool to have and it not bundled in prod version